Airports/EGNO/VRPs.txt

```diff
@@ -1,5 +1,5 @@
 *Garstang Marina:N053.54.19.500:W002.47.30.200
-*M55 J3:N053.48.25.000:W002.53.23.000
 *M6 J30/M61:N053.44.08.100:W002.38.56.100
 *M6 J32/M55:N053.48.27.400:W002.41.58.800
+*M55 J3:N053.48.25.000:W002.53.23.000
```

Comment:
Can we have them (as they were) in alpha-numeric order please? So in essense just leave this file exactly as it was initially :) Ta!

Reply:
Not sure why you marked this as resolved @rishab-alt - it is still in the wrong order and hence showing the EGNO file as changed. Do you need some help doing this? I thought we discussed it and you understood what I meant.